wsd/COOLWSD.cpp

```diff
@@ -3475,6 +3565,22 @@ class ClientRequestDispatcher final : public SimpleSocketHandler
                 const auto pos = uri.find(ProxyRemoteStatic);
                 if (pos != std::string::npos)
                 {
+                    if (Util::endsWith(uri, "lokit-extra-img.svg"))
```

Comment:
This looks dangerous to me - from what I can see, we'll soon end up in a situation where we try to load the rating from an image URI or the other way around ;-)

Let's instead add serverURI as the 1st parameter of the ProxyRequestHandler::handleRequest(), and for the rating server, let's still have a ratingServerProxy constant that we can use there, and let's get rid of the setProxyServer() call.